packages/slate-editor/src/extensions/loader/transforms/replaceLoader.ts

```diff
@@ -12,15 +12,17 @@ export function replaceLoader(editor: Editor, loader: LoaderNode, element: Eleme
 
     const wasSelected = EditorCommands.isTopLevelNodeSelected(editor, loader);
 
-    Transforms.removeNodes(editor, {
-        at: loaderPath,
-        match: (node) => isLoaderElement(node) && node.id === loader.id,
-    });
+    Editor.withoutNormalizing(editor, () => {
+        Transforms.removeNodes(editor, {
+            at: loaderPath,
+            match: (node) => isLoaderElement(node) && node.id === loader.id,
+        });
 
-    Transforms.insertNodes(editor, element, {
-        at: loaderPath,
-        mode: 'highest',
-    });
+        Transforms.insertNodes(editor, element, {
+            at: loaderPath,
+            mode: 'highest',
+        });
 
-    if (wasSelected) Transforms.select(editor, loaderPath);
+        if (wasSelected) Transforms.select(editor, loaderPath);
```

Comment:
maybe at the end we still need to call normalization manually?

Reply:
> maybe at the end we still need to call normalization manually?

Why? 

https://docs.slatejs.org/api/nodes/editor#editor.withoutnormalizing-editor-editor-fn-greater-than-void-greater-than-void

> ```ts
> Editor.withoutNormalizing(editor: Editor, fn: () => void) => void
> ```
> 
> Call a function, deferring normalization until after it completes.